README.md

```diff
@@ -1,5 +1,29 @@
 # Draft topic
 
+Warning: As of late 2023, the ORES infrastructure is being deprecated by the
+WMF  Machine Learning team, please check
+https://wikitech.wikimedia.org/wiki/ORES for more info.
+
+While the code in this repository may still work, it is unmaintained, and as
+such may break at any time. Special consideration should also be given to
+machine learning models seeing drift in quality of predictions.
+
+The replacement for ORES and associated infrastructure is Lift Wing:
+https://wikitech.wikimedia.org/wiki/Machine_Learning/LiftWing
+
+Some Revscoring models from ORES run on the Lift Wing infrastructure, but they
+are otherwise unsupported (no new training or code updates).
+
+They can be downloaded from the links documented at:
+https://wikitech.wikimedia.org/wiki/Machine_Learning/LiftWing#Revscoring_models_(migrated_from_ORES)
+
+In the long term, some or all these models *may* be replaced by newer models
+specifically tailored to be run on modern ML infrastructure like Lift Wing.
+
+If you have any questions, contact the WMF Machine Learning team:
+https://wikitech.wikimedia.org/wiki/Machine_Learning
+
+
```

Comment:
nit: I suggest to add this in a code block with a warning emoji to make it more visible
```suggestion
<blockquote>
⚠️ Warning: As of late 2023, the ORES infrastructure is being deprecated by the
WMF  Machine Learning team, please check
https://wikitech.wikimedia.org/wiki/ORES for more info.

While the code in this repository may still work, it is unmaintained, and as
such may break at any time. Special consideration should also be given to
machine learning models seeing drift in quality of predictions.

The replacement for ORES and associated infrastructure is Lift Wing:
https://wikitech.wikimedia.org/wiki/Machine_Learning/LiftWing

Some Revscoring models from ORES run on the Lift Wing infrastructure, but they
are otherwise unsupported (no new training or code updates).

They can be downloaded from the links documented at:
https://wikitech.wikimedia.org/wiki/Machine_Learning/LiftWing#Revscoring_models_(migrated_from_ORES)

In the long term, some or all these models *may* be replaced by newer models
specifically tailored to be run on modern ML infrastructure like Lift Wing.

If you have any questions, contact the WMF Machine Learning team:
https://wikitech.wikimedia.org/wiki/Machine_Learning
</blockquote>

```

Reply:
Added the warning emoji and blockquote tags; using triple-backticks made the links not clickable, and I'd prefer them to be easily used.